agent/bench-scripts/tests/pbench-user-benchmark/test-67.txt

```diff
@@ -1,5 +1,10 @@
 +++ Running test-67 pbench-user-benchmark
+pbench-user-benchmark: WARNING: No tools are registered
+pbench-start-tools: WARNING: No tools are registered
```

Comment:
I don't understand what changed in this commit (the fourth one) which caused these messages to appear when/if they didn't appear in the previous commit.

However, if you squash this commit with the previous one, then this diff will disappear, and I won't have to raise the point....